sky/packages/sky/lib/src/widgets/basic.dart

```diff
@@ -243,6 +244,28 @@ class Center extends Align {
     : super(key: key, shrinkWrap: shrinkWrap, child: child);
 }
 
+class CustomOneChildLayout extends OneChildRenderObjectWidget {
+  CustomOneChildLayout({
+    Key key,
+    this.delegate,
```

Comment:
document that the delegate should be long-lived since every time it's newed up, we'll relayout.